app/src/main/kotlin/net/corda/cli/application/logger/LoggerStream.kt

```diff
@@ -0,0 +1,68 @@
+package net.corda.cli.application.logger
+
+import java.io.OutputStream
+import java.io.PrintStream
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.io.ByteArrayOutputStream
+
+enum class LogLevel {
+    TRACE, DEBUG, INFO, WARN, ERROR
+}
+
+class LoggerStream(
+    private val logger: Logger,
+    private val logLevel: LogLevel
+) : OutputStream() {
+
+    public companion object {
+        fun redirectSystemAndErrorOut() {
+            val sysOut: Logger = LoggerFactory.getLogger("System Out")
+            val errOut: Logger = LoggerFactory.getLogger("System Err")
+
+            System.setOut(PrintStream(LoggerStream(sysOut, LogLevel.INFO)))
+            System.setErr(PrintStream(LoggerStream(errOut, LogLevel.ERROR)))
+        }
+    }
+
+    private val baos = ByteArrayOutputStream(1000)
+
+    override fun write(b: Int) {
```

Comment:
What's with the variable names? Can we change them to something more descriptive for readability?